src/App.test.tsx

```diff
@@ -1,9 +0,0 @@
-import React from "react";
```

Comment:
Why did you drop this test?

Reply:
This test will have to be replaced with an enzyme based one. But that requires some more work for getting the router and store to work in a testing environment. This PR is meant to just establish a base framework.

Reply:
react-testing-library is an official replacement for enzyme. There's a big blog written by kent c dodds why he discourages enzyme and it's shallow render feature

Reply:
What do you mean by "official"? React doesn't have an official testing suite. Enzyme is still there along with others in the [official react docs](https://reactjs.org/community/testing.html). If you can point me to the react-testing-library docs for typescript, I will more than happy to try it out.

Reply:
> What do you mean by "official"? React doesn't have an official testing suite. Enzyme is still there along with others in the [official react docs](https://reactjs.org/community/testing.html). If you can point me to the react-testing-library docs for typescript, I am more than happy to try it out.

Why do you have to write your tests in that? I guess its not mandatory for writing your tests in typescript

Reply:
I'll push some boillerplate for it

Reply:
> Why do you have to write your tests in that? I guess its not mandatory for writing your tests in typescript

I am not talking about writing tests in TS but testing code written in TS. That's what ts-jest is used for.

Reply:
> I'll push some boillerplate for it

Cool. Thanks!

Reply:
![test](https://user-images.githubusercontent.com/30630904/54866830-cf52ea80-4d9e-11e9-8d27-fa76d5ea269e.PNG)

I don't think something like this would be wrong

Reply:
Can you please share the configs? Maybe push it to a branch and share a link?

Reply:
I'll push this configuration as a PR and will also share my fork with you in a moment

Reply:
@aswinmprabhu here:-
https://github.com/nik72619c/fixme/tree/nik

Reply:
![image](https://user-images.githubusercontent.com/31558262/54875654-dcb4b700-4e28-11e9-85c5-94d8951ca4ea.png)
When I replace `Spinner` with `App`, this is the error that I get. This one of the errors I got when I was setting up react-testing-library. Then I followed the official react-testing-library docs regarding react-router but that didn't work either. A similar error occurs when I try to render `FixMeNavbar` as well.

Reply:
@aswinmprabhu can you check if this is still happening at #134 . Though if the test is a snapshot test then switching the component should break the test. You can read more about snapshots here https://jestjs.io/docs/en/snapshot-testing

Reply:
@aswinmprabhu Could you add this one back? It isn't a huge test but it's important to test that the app doesn't crash